### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: conf/nginx.conf [hunk 1]
```diff
@@ -29,11 +29,6 @@ location __PATH__/UP {
     proxy_set_header        Content-Type application/json;
     proxy_pass            http://127.0.0.1:__PORT__/message;
     proxy_set_header            Host $host;
-
-    # Force https
```
Comment (plopoyop) [inline conf/nginx.conf:33]: I don't realy follow ynh evolutions, do we stop redirecting https ?

Comment (ericgaspar) [reply conf/nginx.conf:33]: this is now on the core...

Comment (plopoyop) [reply conf/nginx.conf:33]: ok, thanks

**Comment by plopoyop** [comment]
lgtm, ok to merge

**Pull request merged**